src/app/docs/country-field/country-field-docs.component.ts

```diff
@@ -1,11 +1,43 @@
 import {
   ChangeDetectionStrategy,
-  Component
+  Component,
+  OnInit
 } from '@angular/core';
 
+import {
+  FormControl,
+  FormGroup
+} from '@angular/forms';
+
+import {
+  SkyCountryFieldCountry
+} from '../../public/modules/country-field/types/country';
+
 @Component({
   selector: 'app-country-field-docs',
   templateUrl: './country-field-docs.component.html',
+  styleUrls: ['./country-field-docs.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class CountryFieldDocsComponent {}
+export class CountryFieldDocsComponent implements OnInit {
+
+  public countryData: SkyCountryFieldCountry;
+
+  public countryForm: FormGroup;
+
+  public countryControl: FormControl;
```

Comment:
Check your alphabetical ordering of properties here ☝️

Reply:
Done